### ISSUE TITLE
Row and column count of a table is not read out by Jaws screen reader while using fd-table

### ISSUE THREAD

**Issue by Nitin-789** [SAP/fundamental-ngx#8401]
#### Is this a bug, enhancement, or feature request?

Bug

#### Briefly describe your proposal.

fd-table consists of cells which are link hence the user can navigate it via tab keypress, so while accessing the first link basically the screen reader should have read the total count of rows and columns but it doesn't reads it out.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

angular - 13
fundamental ngx - 0.34.3-rc.47

#### If this is a bug, please provide steps for reproducing it.

Access the fd-table-cell present inside fd-table-body which in turn is present inside fd-table via tab and observe the screen reader not reading the total count of rows and columns

#### Please provide relevant source code if applicable.

`<table fd-table [attr.awname]="'UAV_TD_6'">
    <thead fd-table-header [attr.awname]="'UAV_TD_7'">
        <tr fd-table-row [attr.awname]="'UAV_TD_8'">
            <th fd-table-cell [attr.awname]="'UAV_TD_9'">{{'UAV.DOCUMENT' | translate}}</th>
        </tr>
    </thead>
    <tbody fd-table-body [attr.awname]="'UAV_TD_11'">
        <tr *ngFor="let row of displayedRows" fd-table-row [attr.awname]="'UAV_TD_12'">
            <td fd-table-cell [attr.awname]="'UAV_TD_12_1_' + row.documentName"><a tabindex="0"
                (click)="downloadTenderDoc(row.documentName, row.documentType);" [attr.awname]="'UAV_TD_12_2_' + row.documentName">{{ row.documentName }}</a></td></tr></tbody></table>`
